### DISCUSSION TITLE
Moms in Tech/WIT  VC meetups

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/VC-Community-Docs#166] [Ideas]
- A place to work through details, prep, etc.
- Let's define audience, location, dates, topic, structure of meetings

@aurelieverrot

**Comment by BekahHW** [comment]
Thing's we've talked about:

- Start with moms
- July 19th at 12pm edt for the first one
- maybe start with a 10-minute presentation and move into a discussion
- Location: zoom, add on meetingplace?
- Maybe we should start a channel

**Reply by aurelieverrot** [reply]
Should we schedule the next date? Is Sept 20th, 12pm eastern working?

**Reply by BekahHW** [reply]
I can get it on the schedule! Should we reach out and see if anyone else is up for speaking? I'll just be coming home from a 3-day conference, so idk if I'm up for speaking that soon.

**Comment by aurelieverrot** [comment]
First MinT (Moms in Tech) meetup was on July 26th, 12pm Eastern.

We talked about Self-Care:
Welcome the group
Talk
Discussion

Feedback: 
- welcome + discussion lasted almost 25 min, might be too long.
- ending the talk with questions to prepare the discussion and avoid too much silence
- the meetup was join by non-mom and dad
- encourage others to give a short talk / share an experience for next meetups.

**Comment by aurelieverrot** [comment]
Next MinT is on September 20th at 12:30p ET
Talk will be done by Cristien about the Working Mom Guilt.
